tests/hooks.py

```diff
@@ -0,0 +1,8 @@
+import re
+from .constants import ACCOUNT_ID
+
+
+class TerraformHooks():
```

Comment:
per previous lets call these rewrite hooks, also rather a separate module let's inline to conftest, the separate module for a five liner seems less needless indirection. also for future ref we want to use standard headers on new code files.